sw/lib/sw/device/silicon_creator/rom_patch.h

```diff
@@ -0,0 +1,86 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_LIB_SW_DEVICE_SILICON_CREATOR_ROM_PATCH_H_
+#define OPENTITAN_SW_LIB_SW_DEVICE_SILICON_CREATOR_ROM_PATCH_H_
+
+#include "sw/lib/sw/device/base/macros.h"
+#include "sw/lib/sw/device/base/multibits.h"
+#include "sw/lib/sw/device/silicon_creator/error.h"
+#include "sw/lib/sw/device/silicon_creator/sigverify/sigverify.h"
+
+#include "rv_core_ibex_regs.h"
+
+/**
+ * A ROM patch address matching table entry.
+ */
+typedef struct rom_patch_match_regs {
+  uint32_t m_base;  // Matching address
+  uint32_t r_base;  // Remapped address
+} rom_patch_match_regs_t;
+
+/**
+ * A ROM patch structure.
+ *
+ * The patch signature is located after the patch code section.
+ */
+typedef struct rom_patch {
+  /**
+   * ROM patch header:
+   *
+   * - Bits 0-3: Lock Valid MuBi
```

Comment:
Is MuBi a typo or have I missed something? If I've missed something, maybe a further comment what that is would be useful?

Reply:
LGTM

Reply:
Thanks!